packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -380,11 +401,8 @@ export class GarbageCollector implements IGarbageCollector {
 			this.runtime.updateTombstonedRoutes(this.tombstones);
 		}
 
-		// Update the summary state tracker's state from this snapshot.
-		this.summaryStateTracker.updateStateFromSnapshotData(snapshotData);
```

Comment:
Remind me why this can go away?

Reply:
Because `summaryStateTracker.initializeBaseState` is called above I guess...?

Reply:
When loading from snapshot, this is done explicitly via `summaryStateTracker.initializeBaseState` when initializing. When refreshing from snapshot, this happens in the `refreshLatestSummary` method in the summary state tracker.
Not updating this in `refreshLatestSummary` and doing it in a separate step was not correct because it added the posibility of adding inconsistent state.